Pipfile

```diff
@@ -27,6 +27,7 @@ retry = "==0.9.2"
 geoip2 = "*"
 bcrypt = "*"
 werkzeug = "==1.0.1"
+isort = "==4.3.0"
```

Comment:
Please move this to dev-packages. Once you do that run `pipenv lock` to build a new Pipfile.lock